### PR TITLE
adding additional type for topic check

### PR DESCRIPTION
**Pull request by przemas75**
Issue https://github.com/jpmens/mqttwarn/issues/358

### CODE REVIEW

File: services/twitter.py [hunk 1]
```diff
@@ -21,13 +21,20 @@ def plugin(srv, item):
...
     try:
         srv.logging.debug("Sending tweet to %s..." % (item.target))
         res = twapi.PostUpdate(text, trim_user=False)
         srv.logging.debug("Successfully sent tweet")
     except twitter.TwitterError, e:
         srv.logging.error("TwitterError: %s" % (str(e)))
         return False
+    except UnicodeDecodeError:
+        srv.logging.error("Your message could not be encoded.")
+    return False
```
Comment (jpmens) [inline services/twitter.py]: are you quite sure this `return False` is correct? (indentation)

Comment (przemas75) [reply services/twitter.py]: Thanks for this catch.

File: samplefuncs.py [hunk 2]
```diff
@@ -9,6 +9,16 @@
     import simplejson as json
 
 def OwnTracksTopic2Data(topic):
+    if type(topic) == unicode:
```
Comment (jpmens) [inline samplefuncs.py]: why is this required, and can you not merge this if together with the one on line 22?

File: services/twitter.py [hunk 3]
```diff
@@ -21,13 +21,20 @@ def plugin(srv, item):
     )
```
Comment (jpmens) [inline services/twitter.py]: it is unfortunate, that you are mixing two different things in one pull request; please try to avoid that in future. (owntracks in samplefuncs.py has nothing to do with twitter.py)

File: services/twitter.py [hunk 4]
```diff
@@ -21,13 +21,20 @@ def plugin(srv, item):
...
 
     text = item.message[0:138]
+    try:
+        text = text.decode('utf-8')
+    except Exception,  err:
+        srv.logging.debug("Cannot convert to unicode: %s" % (str(err)))
```
Comment (jpmens) [inline services/twitter.py]: are you sure this message is correct? that you are converting TO unicode?

Comment (przemas75) [reply services/twitter.py]: I think i understand why you questioning that. I meant converting to unicode string from string which contains non-ascii, it would be 'changing' meaning rather
As i remember 'text' had issues with non-ascii chars as a string.

File: services/twitter.py (hunk 3 above)
Comment (przemas75) [reply services/twitter.py]: Yes, i understand that.

File: samplefuncs.py (hunk 2 above)
Comment (przemas75) [reply samplefuncs.py]: Because in my situation, topic was not type of string but unicode. I have simply checked this.
Im afraid i dont understand this well.
